backend/services/interfaces/classService.ts

```diff
@@ -36,6 +36,7 @@ export interface ClassCard {
   assessmentCount: number;
   gradeLevel: Grade;
   className: string;
+  startDate?: Date;
```

Comment:
why optional?

Reply:
Some of our data in Mongo is missing a `startDate` for some reason, and so I wanted the types to reflect what the front-end actually gets passed